### PR TITLE
tabs height increased

### PR DESCRIPTION
**Pull request by diego-pascual**

### CODE REVIEW

File: src/Tabs/style.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .ola_tabs {
     display: flex;
-    height: var(--size-8);
+    height: calc(var(--size-8)*1.4);
```
Comment (oscarotero) [inline src/Tabs/style.css:3]: Using random values is agains the style guide. Why do you want to increase the height?

Comment (diego-pascual) [reply src/Tabs/style.css:3]: in the design it has a higher height, --size-8 is small and --size-9 is too high. How can I do it? @oscarotero

Comment (diego-pascual) [reply src/Tabs/style.css:3]: It's not important, we can left in --size-8 and I can add a padding top to tabs. forget it

Comment (oscarotero) [reply src/Tabs/style.css:3]: Ok, understood.
I think the problem is you've used a ola_button for that.
Maybe something like `.ola_menu-option` is better (https://github.com/marketgoo/Ola/blob/development/src/Menu/style.css#L6) (look like it doesn't have a fixed width but a padding).

You should create a `ola_tabs-option` and apply some similar styles.

Comment (diego-pascual) [reply src/Tabs/style.css]: Yeah, it's better option, I changed it, please, check when you can @oscarotero

**Review by oscarotero — APPROVED** [review]

**Pull request merged**